site/assets/scss/_component-examples.scss

```diff
@@ -359,10 +359,11 @@
   }
 
   pre {
-    padding: 0;
-    margin-top: .625rem;
+    padding: .2rem 0 .9rem;
```

Comment:
I'd echo @romaricpascal's question here—let's go for whole numbers on the computed values. We do this throughout.

```suggestion
    padding: .25rem 0 .875rem;
```